### PR TITLE
Add Ricoh PPD v3.4.0.0

### PR DESCRIPTION
**Pull request by PetroccoCo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-drivers/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/ricoh-ppd.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask 'ricoh-ppd' do
```
Comment (reitermarkus) [inline Casks/ricoh-ppd.rb]: This must be be named according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/43d6f965c3d03393ae01935c22966c0f802e44a3/doc/cask_language_reference/token_reference.md), based on `Ricoh_PS_Printers_Vol4_EXP_LIO_Driver.pkg`.

File: Casks/ricoh-ppd.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  version '3.4.0.0'
+  sha256 'ffd01b48ef1d3e0b86894eda316068b799178969075929544c7a832d50f9bf9a'
+
+  url "http://support.ricoh.com/bb/pub_e/dr_ut_e/0001310/0001310123/V#{version.delete('.')}/Ricoh_PS_Printers_Vol4_EXP_LIO_#{version}.dmg"
+  name 'Ricoh PPD Installer'
```
Comment (reitermarkus) [inline Casks/ricoh-ppd.rb]: This should also be named based on `Ricoh_PS_Printers_Vol4_EXP_LIO_Driver.pkg`.

**Comment by PetroccoCo** [comment]
No problem, perhaps something like "ricoh-psprintersexp" or "ricoh-psprinters"?

My thoughts were Vol4 is a version that should be stripped.
LIO appears to refer to Linux IO which in my mind is a hardware designation.
EXP is a model classifier for Ricoh so that could arguably stay.

FYI Brew's token generator gives:
````
"$(brew --repository)/Library/Taps/Homebrew/homebrew-cask/developer/bin/generate_cask_token" /usr/local/Caskroom/ricoh-ppd/3.4.0.0/Ricoh_PS_Printers_Vol4_EXP_LIO_Driver.pkg
Proposed token:               ricohpsprintersvol4expliodriverpkg
Proposed file name:           ricohpsprintersvol4expliodriverpkg.rb
Cask Header Line:             cask 'ricohpsprintersvol4expliodriverpkg' do

WARNING: 'ricohpsprintersvol4expliodriverpkg' contains digits. Digits which are version numbers should be removed.
````

**Comment by reitermarkus** [comment]
We already have `ricoh-driver-aficio-sp-c240dn` (which I think should be renamed to `ricoh-aficio-sp-c240dn-driver`), so maybe `ricoh-ps-printers-exp-driver`.

**Comment by reitermarkus** [comment]
Also, I get a different URL from the homepage:

`http://support.ricoh.com/bb/pub_e/dr_ut_e/0001310/0001310159/V3400/Ricoh_PS_Printers_Vol3_EXP_LIO_3.4.0.0.dmg`

File: Casks/ricoh-ppd.rb (hunk 1 above)
Comment (PetroccoCo) [reply Casks/ricoh-ppd.rb]:
```suggestion
cask 'ricoh-ps-printers-exp-driver' do
```

File: Casks/ricoh-ppd.rb (hunk 2 above)
Comment (PetroccoCo) [reply Casks/ricoh-ppd.rb]:
```suggestion
  name 'Ricoh PS printers EXP driver'
```

**Comment by PetroccoCo** [comment]
Also in my uninstall, the two will uninstall pkgs used by each other, they don't directly conflict with each other though so I didn't think a caveats or conflicts-with stanza was appropriate.

**Comment by reitermarkus** [comment]
Please split this up into two separate PRs.

**Comment by PetroccoCo** [comment]
Split into pull requests:
https://github.com/Homebrew/homebrew-cask-drivers/pull/715
https://github.com/Homebrew/homebrew-cask-drivers/pull/714

**Pull request closed without merging**